src/opnsense/mvc/app/controllers/OPNsense/Firewall/Api/FilterController.php

```diff
@@ -389,21 +388,52 @@ public function getInterfaceListAction()
             ]
         ];
 
-        foreach ((new Group())->ifgroupentry->iterateItems() as $groupItem) {
-            $groupName = (string)$groupItem->ifname;
-            $result['groups']['items'][$groupName] = ['value' => $groupName, 'label' => $groupName];
+        // Count rules per interface
+        $ruleCounts = [];
+        foreach ((new \OPNsense\Firewall\Filter())->rules->rule->iterateItems() as $rule) {
+            $interfaces = array_filter(explode(',', (string)$rule->interface));
+
+            if (count($interfaces) !== 1) {
+                // floating: empty or multiple interfaces
+                $ruleCounts[''] = ($ruleCounts[''] ?? 0) + 1;
```

Comment:
indexing by empty string is a bit weird, may improve readability if we just call it `floating`